src/App.scss

```diff
@@ -22,17 +23,33 @@
 }
 
 .soon {
-  background-color: green;
+  background-color: #ff776e;
 }
 
 .kinda-soon {
-  background-color: yellow;
+  background-color: #fdab6d;
 }
 
 .not-soon {
-  background-color: red;
+  background-color: #fee0b3;
 }
```

Comment:
I would add all the colours as variables in `variables.scss` to make it easier to update in the future if we wanted to change the colours 😃

Reply:
Agreed!! I had just filled these in initially with random colors to satisfy the old issue but now we can make use of sass